src/System.IO.IsolatedStorage/tests/System/IO/IsolatedStorage/ContainsUnknownFilesTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using Xunit;
+
+namespace System.IO.IsolatedStorage
+{
+    public class ContainsUnknownFilesTests : IsoStorageTest
+    {
+        private static MethodInfo s_containsUnknownFilesMethod;
+
+        static ContainsUnknownFilesTests()
+        {
+            s_containsUnknownFilesMethod = typeof(IsolatedStorageFile).GetMethod("ContainsUnknownFiles", BindingFlags.NonPublic | BindingFlags.Instance);
+        }
+
+        [Theory MemberData("ValidStores")]
+        public void ContainsUnknownFiles_CleanStore(PresetScopes scope)
+        {
+            TestHelper.WipeStores();
```

Comment:
Most tests don't care. I want to leave the overall state clean at the end of tests (fixture is the only way to do this without writing custom runners) but not wipe everything every single test / test class.